build/webpack/webpack.services.js

```diff
@@ -31,7 +31,8 @@ config.plugins.push(new CopyWebpackPlugin(
 	[
 		{
 			from: './src/services/',
-			to: './services/'
+			to: './services/',
+			ignore: ['*.ts', '*.js', '*.jsx', '*.tsx'],
```

Comment:
This file can be left out if needed, it clears up some of the duplicate files created by webpack but I'm more concerned about it causing any unwanted side effect, needs looking at.